### DISCUSSION TITLE
Q: Supported quantizations for MacOS Metal ?

### DISCUSSION THREAD

**Discussion by ianscrivener** [ggml-org/llama.cpp#2320] [Q&A] [answered]
Previsouly only q4_0 and FP16 were supported... is that still the case?

**Comment by ggerganov** [comment, marked as answer]
The supported quantizations by Metal are listed in the following `switch`:

https://github.com/ggerganov/llama.cpp/blob/5d500e8ccf5eee3de3ae66685cc3be75e43e08b9/ggml-metal.m#L646-L724

**Reply by ianscrivener** [reply]
thanks @ggerganov

**Comment by ianscrivener** [comment]
For the lazy folks:
  F16
  Q6_K
  Q5_K
  Q4_K
  Q4_1
  Q4_0
  Q3_K
  Q2_K
